wwwapp/models.py

```diff
@@ -59,6 +59,28 @@ class WorkshopUserProfile(models.Model):
     def __str__(self):
         return '%s: %s, %s' % (self.year, self.user_profile, self.status)
 
+class PESELField(models.CharField):
+    """PESEL field, with checksum verification."""
+
+    def clean(self, *args, **kwargs):
+        pesel = args[0] 
+
+        # https://en.wikipedia.org/wiki/PESEL#Format
+
+        if len(pesel) != 11:
```

Comment:
PESEL may be None. len(None) will crash with an TypeError.

Reply:
If I am not mistaken, this can't happen, as there is a default value of empty string.

Reply:
I checked the code, @enedil is right here - if the value was `None`, it will be coerced to empty_value (default `''`) before it reaches the `validate` method.